### ISSUE TITLE
macOS: Missing SDL2 in app bundle

### ISSUE THREAD

**Issue by shinra-electric** [atsb/Hiigara#3]
The app cannot launch without SDL2 installed in the recommended places (`/opt/local/include` and `/opt/local/lib`). 

```
`dyld[18463]: Library not loaded: @rpath/SDL2.framework/Versions/A/SDL2
  Referenced from: <45693886-7F20-3067-B6E5-427581F8DFFE> /Users/user/Downloads/Homeworld.app/Contents/MacOS/Homeworld
  Reason: tried: '/Users/user/Downloads/Homeworld.app/Contents/MacOS/../Frameworks/SDL2.framework/Versions/A/SDL2' (no such file), '/Users/user/Downloads/Homeworld.app/Contents/MacOS/../Frameworks/SDL2.framework/Versions/A/SDL2' (no such file), '/Library/Frameworks/SDL2.framework/Versions/A/SDL2' (no such file), '/System/Library/Frameworks/SDL2.framework/Versions/A/SDL2' (no such file, not in dyld cache)
[1]    18463 abort      ./Homeworld
```

The app bundle should come with all required libraries, with an SDL2 dylib in the `Frameworks` directory. 

(Unsure if related, but putting the `SDL2.framework` folder in `Frameworks` silences this error, but the app crashes immediately on launch still)

**Comment by atsb** [comment]
I have fixed the project now.  I also uploaded a new binary built from the new project.

**Comment by shinra-electric** [comment]
Fixed with the 1.1 update
